### ISSUE TITLE
Add example

### ISSUE THREAD

**Issue by lthibault** [al8n/memberlist#41]
Does anyone have a minimal example of using memberlist?

If someone is able to help me get started, I'll gladly contribute an example to the repo.

Thanks!

**Comment by al8n** [comment]
> Does anyone have a minimal example of using memberlist?
> 
> 
> 
> If someone is able to help me get started, I'll gladly contribute an example to the repo.
> 
> 
> 
> Thanks!

Yes, feel free to do that. PRs are welcome. Actually, there is https://github.com/al8n/ruserf which is built on top of memberlist, you can see the unittests2 branch, which is almost ready for first release.

**Comment by lthibault** [comment]
Thank you!  Looking now :)

**Comment by al8n** [comment]
A simple example is available now in [`examples/toydb`](https://github.com/al8n/memberlist/tree/main/examples/toydb). However, version `0.6` has not been released yet.

**Comment by al8n** [comment]
Closed this issue. `0.6` has been published, and a production project based on the `memberlist` is also available [here](https://github.com/al8n/serf).

A Discord server is also available [here](https://discord.gg/f3EtWBM7ke)
